library/systemd/src/modules/Systemd.rb

```diff
@@ -38,7 +38,7 @@ def main
       @systemd_path = "/usr/lib/systemd/systemd"
       @default_target_symlink = "/etc/systemd/system/default.target"
       @systemd_targets_dir = "/usr/lib/systemd/system"
-      @systemd_mountdir = "/sys/fs/cgroup/systemd"
+      @running_dir = "/run/systemd"
```

Comment:
is it needed without alternative?

Reply:
No, it is not, we can remove it if we are fine with the current solution